galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/internal/CicstsDefaultLogonProvider.java

```diff
@@ -52,10 +65,38 @@ public boolean logonToCicsRegion(ICicsTerminal cicsTerminal) throws CicstsManage
 
             waitForGmText(cicsTerminal);
 
-            cicsTerminal.clear();
-
             logger.debug("Logged onto " + cicsTerminal.getCicsRegion());
-        } catch (Zos3270Exception e) {
+
+            // If loginCredentials is provided, attempt to sign-in
+            // via CESL
+            if (!cicsTerminal.getLoginCredentials().isEmpty()) {
```

Comment:
if the cicsTerminal creds cannot be found, we might want to consider getting the z/OS image that the CICS is running on and using those creds.

Reply:
@hobbit1983 I think my concern is the same as my other comment about defaulting to logging in via a particular credential. Logging in via CESL doesn't happen by default for users at the moment, so they might have their own code that logs in within their tests etc.
If I make this change, logging in via CESL will happen by default. I don't know if it will make any difference to existing tests for people? Colin also had concerns about which user is used as there's security concerns if its a particular one, possibly the default user (he'd need to confirm).
If someone can confirm my worries aren't relevant, then I'll make the change. should we perhaps have a meeting with Colin?

Reply:
Agreed with @hobbit1983 & Colin in a meeting that if `loginCredentialsTag` isn't provided, CESL won't be attempted.